default.toml

```diff
@@ -0,0 +1,18 @@
+max_width = 120
+ideal_width = 100
+leeway = 5
+tab_spaces = 4
+newline_style = "Unix"
+fn_brace_style = "SameLineWhere"
+fn_return_indent = "WithArgs"
+fn_args_paren_newline = true
+struct_trailing_comma = "Vertical"
+struct_lit_style = "BlockIndent"
+struct_lit_trailing_comma = "Vertical"
+enum_trailing_comma = true
+report_todo = "Always"
+report_fixme = "Never"
+reorder_imports = false
+expr_indent_style = "Tabbed"
+closure_indent_style = "Visual"
+single_line_if_else = false
```

Comment:
are the settings in this file taken from a "standard" defaults or based on anything?